src/Impostor.Server/Net/Inner/Objects/InnerMeetingHud.cs

```diff
@@ -168,6 +168,21 @@ public override void Deserialize(IClientPlayer sender, IClientPlayer? target, IM
                     if ((num & 1 << i) != 0)
                     {
                         _playerStates[i].Deserialize(reader);
+                        var playerState = _playerStates[i];
+
+                        if (playerState.DidVote)
```

Comment:
Also trigger the event if the player didn't vote, and set VoteType to None.

Reply:
This would allow plugins to avoid writing logic to figure out who didn't vote